optaplanner-core/src/test/java/org/optaplanner/core/api/score/stream/uni/UniConstraintStreamTest.java

```diff
@@ -1388,7 +1388,7 @@ public void globalNodeOrder() {
                     .filter(entity -> entity.getEntityGroup() == entityGroup)
                     .filter(entity -> entity.getStringProperty().equals("MyString1"))
                     .join(TestdataLavishEntity.class, equal(TestdataLavishEntity::getIntegerProperty))
-                    .penalize(TEST_CONSTRAINT_NAME, SimpleScore.ONE);
+                    .penalize(TEST_CONSTRAINT_NAME, SimpleScore.ONE, (a, b) -> 1);
```

Comment:
Seems wierd, because the argument returns "1", so it's impact isn't asserted. Does it work without that extra argument?

Reply:
That's a leftover from debugging. Yes, it works, will remove.